apps/mobile/src/features/approver/components/approver-account-card.tsx

```diff
@@ -0,0 +1,27 @@
+import { AvatarIcon } from '@/components/avatar-icon';
+import { AccountListItem } from '@/features/account-list/account-list-item';
+import { useWallets } from '@/store/wallets/wallets.read';
+import { t } from '@lingui/macro';
+import { useTheme } from '@shopify/restyle';
+
+import { Text, Theme } from '@leather.io/ui/native';
+
+export function ApproverAccountCard() {
+  const theme = useTheme<Theme>();
+  const { list: walletsList } = useWallets();
+  const wallet = walletsList[0];
+  /* eslint-disable-next-line lingui/no-unlocalized-strings  */
+  const testAccountName = 'testName';
```

Comment:
Maybe just load the first account from wallets here? Then it will be more realistic